fpga_interchange/fpga_interchange.cpp

```diff
@@ -535,11 +535,22 @@ void FpgaInterchange::write_physical_netlist(const Context * ctx, const std::str
         phys_cell.setPhysType(PhysicalNetlist::PhysNetlist::PhysCellType::PORT);
     }
 
-    auto nets = phys_netlist.initPhysNets(ctx->nets.size());
+    int nets_to_remove = 0;
+    for(auto & net_pair : ctx->nets) {
+        auto &net = *net_pair.second;
+
+        if (net.users.size() == 0)
```

Comment:
nitpick `net.users.empty()` would be neater

Reply:
Right, fixed. Thanks